packages/n/nvidia-glx-driver/package.yml

```diff
@@ -61,22 +61,25 @@ patterns   :
     - modaliases :
         - /usr/share/doflicky/modaliases
         - /usr/share/linux-driver-management/modaliases
-    - 32bit : /usr/lib32
+    - 32bit :
+        - /usr/lib32
+        - /usr/share/metainfo/nvidia-32bit.metainfo.xml
     - main :
         - /lib/modules/*.lts
-        - /usr/share/metainfo/nvidia-lts.appdata.xml
-        - /usr/share/pixmaps/nvidia-lts.png
+        - /usr/share/metainfo/nvidia-lts.metainfo.xml
+        - /usr/share/icons/hicolor/128x128/apps/nvidia-lts.png
     - current :
         - /lib/modules/*.current
-        - /usr/share/metainfo/nvidia-current.appdata.xml
-        - /usr/share/pixmaps/nvidia-current.png
+        - /usr/share/metainfo/nvidia-current.metainfo.xml
+        - /usr/share/icons/hicolor/128x128/apps/nvidia-current.png
```

Comment:
The icon should be in `common` as just `/usr/share/icons/hicolor/128x128/apps/nvidia.png`

Reply:
Appstream can only read assets from the current package